packages/select/src/StateFeedback/StateFeedback.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { Error, useUpdatedBaseFontSize } from '@leafygreen-ui/typography';
+
+import { LGIDS_SELECT } from '../constants';
+import { State } from '../Select/Select.types';
+
+import {
+  containerStyles,
+  errorIconThemeStyle,
+  errorTextMobileStyles,
+  hideContainerStyle,
+  validIconThemeStyle,
+  validMessageThemeStyle,
+  validMessageTypeScaleStyles,
+} from './StateFeedback.styles';
+import { StateFeedbackProps } from './StateFeedback.types';
+
+export const StateFeedback = ({
```

Comment:
Is this unique to Select? Could this move to `FormField?

Reply:
moved :)